examples/01_prepare_data/mind_utils.ipynb

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
<p>can you please move all imports to the first cell? In the rest of the notebooks we follow that convention</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>